README.md

```diff
@@ -373,7 +373,7 @@ code eligible for refactoring.
 **Bad:**
 ```javascript
 function showDeveloperList(developers) {
-  developers.forEach(developers => {
+  developers.forEach((developers) => {
```

Comment:
There was another PR about this that got lost in the fray, can we change this to `developers.forEach((developer) =>`